core/src/main/java/com/linecorp/armeria/client/endpoint/healthcheck/HealthCheckedEndpointGroup.java

```diff
@@ -136,17 +140,27 @@ public static HealthCheckedEndpointGroupBuilder builder(EndpointGroup delegate,
     }
 
     private void updateCandidates(List<Endpoint> candidates) {
+        healthCheckStrategy.updateCandidate(candidates);
+        refreshCandidates();
+    }
+
+    private void refreshCandidates() {
```

Comment:
`updateCandidates` and `refreshCandidates` look like they do something similar. Could you rename them so that their responsibility is represented more clearly?

Reply:
Fixed(But not sure, `refreshContexts()` is ok?)